time sensor_ts
time db_ts
float32 temp
float32 hum
float32 pres
float64 co
float64 no
float64 so2
float64 pm1
float64 pm2_5
float64 pm10
